modules/node/security-groups.tf

```diff
@@ -0,0 +1,61 @@
+locals {
+  instanceTcpPorts = ["${var.ssh_port}", 80, 433, 7946]
+  instanceUdpPorts = [7946, 4789]
+
+  managerTcpPorts  = ["${var.ssh_port}", 80, 433, 2377, 7946]
+  managerUdpPorts  = [7946, 4789]
+}
+
+resource "scaleway_security_group" "swarm_instance" {
+  name        = "swarm_instances"
+  description = "Allow SSH, HTTP(S) and internal Swarm traffic"
+
+  inbound_default_policy  = "drop"
+  outbound_default_policy = "accept"
+  stateful                = true
+
+  dynamic "inbound_rule" {
+    for_each = local.tcpPorts
+    content {
+      action    = "accept"
+      port      = inbound_rule.value
+      protocol  = "TCP"
+    }
+  }
+
+  dynamic "inbound_rule" {
+    for_each = local.udpPorts
+    content {
+      action    = "accept"
+      port      = inbound_rule.value
+      protocol  = "UDP"
+    }
+  }
+}
+
+resource "scaleway_security_group" "swarm_manager" {
```

Comment:
The resource was renamed to `scaleway_instance_security_group`

Reply:
Aha, thanks!